server/src/handlers/influxdb.rs

```diff
@@ -311,6 +311,8 @@ impl InfluxqlResultBuilder {
             }
         );
 
+        // Query like `show measurements`, there will be not timestamp.
+        let has_timestamp = column_schemas.iter().any(|c| c.data_type.is_timestamp());
```

Comment:
Name it `is_select`? We just use the existence of `timestamp` to get the query type?

Reply:
Not make much sense.